Tests/test_attr.py

```diff
@@ -56,7 +56,7 @@ def CheckModule(mod):
             CheckDictionary(mod.__dict__)
 
             mod.__dict__[1] = '1'
-            self.assertEqual(dir(mod).__contains__(1), True)
+            self.assertRaises(TypeError, lambda: dir(mod).__contains__(1))
```

Comment:
This one is interesting and kinda makes me wonder whether this is intended behavior, but in Python 3.6, I got:
```
>>> import fractions
>>> fractions.__dict__[1] = '1'
>>> dir(fractions)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'int' and 'str'
```

Reply:
Presumably `dir` does a sort so this would be the expected behavior (e.g. `["a", 1].sort()`).